packages/storybook/.babelrc

```diff
@@ -11,7 +11,25 @@
     "react",
     "stage-2"
   ],
-  "plugins": [
-    "transform-object-rest-spread"
-  ]
+  "plugins": ["transform-object-rest-spread"],
+  "env": {
+    "test": {
+      "presets": [
+        [
+          "env",
+          {
+            "node": "current"
+          }
+        ],
+        "react",
+        "stage-2"
+      ]
+    },
+    "production": {
```

Comment:
Also unclear if these are needed, but we'll pick apart the different babel configs soon.